mailbox/store/src/test/java/org/apache/james/mailbox/store/StoreMailboxManagerTest.java

```diff
@@ -176,8 +177,7 @@ public void getPathLikeShouldReturnUserPathLikeWhenNoBaseDefined() throws Except
         //Given
         MailboxSession session = new MockMailboxSession("user");
         MailboxQuery.Builder testee = MailboxQuery.builder()
-            .expression("abc")
-            .mailboxSession(session);
+            .expression(new PrefixedRegex("", "abc", session.getPathDelimiter()));
```

Comment:
what is this empty string? can you name it?

Reply:
No prefix, regex only ;-)